tests/integration/conftest.py

```diff
@@ -76,7 +77,7 @@ async def app(ops_test: OpsTest, app_name: str, app_config: Dict[str, str], pyte
     await ops_test.model.wait_for_idle()
 
     # Add required relations
-    assert ops_test.model.applications[app_name].units[0].workload_status == WaitingStatus.name  # type: ignore
+    assert ops_test.model.applications[app_name].units[0].workload_status == WaitingStatus.name  # type: ignore # pylint: disable=line-too-long
```

Comment:
This can also be resolved using:

```Python
unit = ops_test.model.applications[app_name].units[0]
assert unit.workload_status == WaitingStatus.name
```

Reply:
Thanks, it is better that way!